src/main.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+class Todo  {
+    constructor() {
+        this.todoPoints = [];
+    }
+
+    addItem(title, priority) {
+        let todoPoint = {};
+        todoPoint.title = title;
+        todoPoint.priority = priority;
+        todoPoint.id = +Math.random().toString().substr(2, 4);
+        this.todoPoints.push(todoPoint);
+        return todoPoint.id
+    }
+
+    getItem(id) {
+        let item = this.todoPoints.filter(item => item.id === id);
+        if (item.length <= 0) {
+            return null;
+        } else {
+            return item;
+        }
+    }
+
+    removeItem(id) {
+        let item = this.todoPoints.filter(item => item.id === id);
+        this.todoPoints = this.todoPoints.filter(item => item.id !== id);
+        return item.length > 0;
+    }
+
+    next() {
+        let highestPriority = Math.max.apply(Math, this.todoPoints.map(item => {return item.priority}))
+        let item = this.todoPoints.find(item => item.priority === highestPriority);
+        if (item.length <= 0) {
```

Comment:
you need to check `todoPoints.length` instead of `item.length`
`...if there are no items in the list...`
and move this checking to the top of this method.
1. we check `todoPoints.length` if it has elements, we move on
2. return priority item